src/components/AppHeader/UserTools/UserMenuButton.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+
+import Button from "../../common/Button";
+import UserMenuDropdown from "./UserMenuDropdown";
+
+import VARIANTS from "../../../constants/variants";
+
+const { ICON } = VARIANTS;
+
+export default function UserMenuButton({ onClick }) {
+  const [toggleDropDown, setToggleDropDown] = useState(false);
+
+  const UserMenuIcon = <img alt="사용자 메뉴 아이콘" src="/images/user_menu_icon.png" width="40" height="40" />;
+
+  const handleDropDown = () => {
+    setToggleDropDown(!toggleDropDown);
+  };
+
+  const handleLoginStatus = (boolean) => {
+    onClick(boolean);
+  };
+
+  return (
+    <>
+      <Button variant={ICON} children={UserMenuIcon} onClick={handleDropDown} />
```

Comment:
Button 컴포넌트에 정의된 props 순서에 맞게 작성해 주세요.

```suggestion
      <Button variant={ICON} onClick={handleDropDown} children={UserMenuIcon} />
```